conf/SiteDefs.pm

```diff
@@ -200,6 +200,7 @@ our @ENSEMBL_CONF_DIRS               = ("$ENSEMBL_WEBROOT/conf");
 our @ENSEMBL_HTDOCS_DIRS             = ($ENSEMBL_DOCROOT, "$ENSEMBL_SERVERROOT/biomart-perl/htdocs");  # locates static content
 our $ENSEMBL_TAXONOMY_DIVISION_FILE  = ("$ENSEMBL_DOCROOT/e_divisions.json");
 our $DEFAULT_SPECIES_IMG_DIR         = 'htdocs/i/species';
+our $SPECIES_IMAGE_DIR               = defer { $SiteDefs::ENSEMBL_SERVERROOT.'/public-plugins/ensembl/'. $SiteDefs::DEFAULT_SPECIES_IMG_DIR };
```

Comment:
I think we dont need to do this. I have seen a PR in public_plugins from Manoj which got approved. So I will update this PR. But please do check the remaining ones.

Reply:
Yes, this needs to go into public-plugins, since many sites won't have this file location available.